packages/php/email-editor/src/Engine/class-email-api-controller.php

```
@@ -90,9 +90,14 @@ public function send_preview_email_data( WP_REST_Request $request ): WP_REST_Res
 	/**
 	 * Returns all registered personalization tags.
 	 *
+	 * @param WP_REST_Request $request Route request parameters.
 	 * @return WP_REST_Response
+	 * @phpstan-param WP_REST_Request<array{_locale: string, email: string, postId: int}> $request
 	 */
-	public function get_personalization_tags(): WP_REST_Response {
+	public function get_personalization_tags( WP_REST_Request $request ): WP_REST_Response {
+		$data      = $request->get_params();
+		$post_type = $data['postType'] ?? null;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$post_type'. (Unused Code Rules)

(UnusedLocalVariable)